CSS/hintwork-style.css

```diff
@@ -0,0 +1,217 @@
+.work-hint-modal-container {
+  position: fixed;
+  top: 5%;
+  right: 1%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  opacity: 0;
+  pointer-events: none;
+  transition: opacity 0.5s ease-in-out;
+  z-index: 10;
+  background-color: #fff;
+  padding-top: 30px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.work-hint-modal-container.active {
+  opacity: 1;
+  pointer-events: auto;
+}
+
+.work-hint-modal {
+  width: 100%;
+  margin: 0 5px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+  position: relative;
+}
+
+.work-hint-modal h2 {
+  display: inline;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 44px;
+  color: #172b4d;
+  letter-spacing: 0.8px;
+  margin: 16px;
+}
+
+.work-hint-modal button {
+  border: none;
+  cursor: pointer;
+  font-size: 30px;
+  font-weight: bold;
+  position: absolute;
+  top: 10px;
+  right: 20px;
+  opacity: 0.3;
+  background: none;
+}
+
+.work-hint-modal button:hover {
+  opacity: 1;
+}
+
+.work-hint-modal ul {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: 16px;
+}
+
+.work-hint-modal li {
+  font-family: 'Poppins', sans-serif;
+  background: #fff;
+  border: 1px solid #979493;
+  padding: 10px 12px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+  color: #3c3a39;
+  margin-right: 16px;
+  list-style: none;
+}
+
+.work-hint-placeholder {
+  width: auto;
+  height: auto;
+  margin: 0 16px;
+}
+
+.work-hint-modal p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  opacity: 0.8;
+  margin: 0 16px;
+}
+
+.d-links {
+  margin: 0 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 90%;
+  padding-bottom: 20px;
+}
+
+.d-links a {
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #edecff;
+  text-align: center;
+  text-decoration: none;
+  border: none;
+  font-size: 14px;
+  line-height: 24px;
+  width: 80%;
+  height: 48px;
+  letter-spacing: 1.3px;
+  padding: 8px;
+}
+
+.d-links a span {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 30vw;
+  margin: 3px;
+}
+
+.work-hint-modal .work-hint-title2 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .work-hint-modal-container {
```

Comment:
In order to respect Figma design, kindly, adapt your display to imitate it as close as possible for the desktop version:

- [ ] links should be aligned to the left, near the image;
- [ ] the image should be bigger in size and should take more than half of the width of the container;
- [ ] the distance between links shouldn't be so big;

Your design:
![image](https://user-images.githubusercontent.com/83596135/162238543-54d603a0-158b-4dcd-bf2c-2894448a7410.png)

Figma design:
![image](https://user-images.githubusercontent.com/83596135/162238729-d714a673-e9d2-4607-9b3c-12f4af9897ec.png)